lib/templates/baseLayout.mustache

```diff
@@ -1,319 +1,578 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <title>Swagger Accuracy Report:{{package}}({{ apiVersion }})</title>
-        <meta charset="UTF-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1">
 
-        <script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
-        <style type="text/css">
-            body {
-                margin: 40px auto;
-                max-width: 900px;
-                line-height: 1.6;
-                font-size: 14pt;
-                color: #444;
-                padding: 0 10px;
-                font-family: Calibri;
-            }
-        
-            h1, h2, h3 {
-                line-height: 1.2;
-                font-family: Arial;
-            }
-        
-            h1 {
-                border-bottom: 1px solid #43425A;
-                width: 100%;
-                background-color: #F6FFFF
-            }
-        
-            h2 {
-                border-bottom: 1px solid lightgrey;
-                background-color: #F6FFFF;
-            }
-        
-            h3 {
-                color: #707070;
-            }
+<head>
+    <title>Swagger Accuracy Report:{{package}}({{ apiVersion }})</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="keywords" content="Coverage, Swagger, Report, Tools">
+    <meta name="description" content="Swagger Accuracy Report">
+    <script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
+    <style>
+      :root {
+        --report-bg-green: #91cc74;
+        --report-bg-red: #ee6666;
+        --report-bg-yellow: #fac858;
+        --report-bg-blue: #e6e9f0;
+        --report-color-green: #218838;
+        --report-color-yellow: #e0a800;
+        --report-color-red: #c82333;
+        --report-base-color: #333;
+        --border-color: #b7b7b7;
+        --report-color-black: #000;
+        --report-color-white: #fff;
+        --title-small: #707070;
+        --link-color: #1453ff;
+        --link-hover-color: #7096ff;
+        --content-hover-color: #f5f5f5;
+        --line-color: #43425a;
+      }
+      .report-content {
+        margin: 10px auto 40px;
+        min-width: 900px;
+        max-width: 1500px;
+        color: var(--report-base-color);
+        font-family: 'Helvetica Neue', Helvetica, Arial, 'Hiragino Sans GB',
+          'Microsoft YaHei UI', sans-serif;
+        box-sizing: border-box;
+      }
+      h2 {
+        margin-bottom: 14px;
+      }
+      a {
+        text-decoration: none;
+        position: relative;
+        display: inline-block;
+        color: var(--link-color);
+      }
+      a span {
+        margin: auto;
+        display: inline-block;
```

Comment:
I tried to render and needs to remove {display:.; position:.; background:.} style of `a span`. Can you check?